mycore-base/src/main/java/org/mycore/common/xml/MCRXMLFunctions.java

```diff
@@ -659,6 +659,21 @@ public static String getDisplayName(String classificationId, String categoryId)
     }
 
     public static boolean isCategoryID(String classificationId, String categoryId) {
+        if (classificationId == null || classificationId.isEmpty()) {
+            LOGGER.error("Classification identifier is null or empty");
```

Comment:
I would reduce all log level to info() or debug() because we "expect" an error here and do not want to pollute the log files.